interfaces/background-fetch.idl

```diff
@@ -19,26 +19,17 @@ interface BackgroundFetchManager {
   Promise<BackgroundFetchRegistration> fetch(DOMString id, (RequestInfo or sequence<RequestInfo>) requests, optional BackgroundFetchOptions options);
   Promise<BackgroundFetchRegistration?> get(DOMString id);
   Promise<sequence<DOMString>> getIds();
-  // TODO: in future this should become an async iterator for BackgroundFetchRegistration objects
 };
 
 dictionary BackgroundFetchUIOptions {
-  sequence<IconDefinition> icons = [];
+  sequence<ImageResource> icons = [];
```

Comment:
ImageResource is defined in appmanifest.idl, but this change doesn't require bringing it in as a dependency, tested locally and saw no changes compared to previous state.